cmake/std/trilinosprhelpers/unittests/trilinos_pr_test.ini

```diff
@@ -306,6 +307,21 @@ unsetenv PATH_TMP:
 
 
 
+[Trilinos_pullrequest_cuda_10.1.105_uvm]
+module-load git: 2.10.1
+module-load devpack: 20190404/openmpi/4.0.1/gcc/7.2.0/cuda/10.1.105
+module-swap openblas/0.3.4/gcc/7.4.0: netlib/3.8.0/gcc/7.2.0
+setenv OMPI_CXX: ${WORKSPACE}/Trilinos/packages/kokkos/bin/nvcc_wrapper
+setenv OMPI_CC: ${CC}
+setenv OMPI_FC: ${FC}
+setenv CUDA_LAUNCH_BLOCKING: 1
```

Comment:
Why is launch blocking on?

Reply:
I started with a copy of the current CUDA 10.1.105 build. We can turn it off.